package.json

```diff
@@ -22,18 +22,14 @@
     "fs-default-project-config": "^6.1.2"
   },
   "peerDependencies": {
-    "postcss-scss": "^4.0.2",
     "stylelint": "^14.0.1"
```

Comment:
i think move it from `peerDependencies` to `dependencies` (which is more inline with how we've done eslint)

then we can remove it from consuming repos too

Reply:
yeah done :)